sidebars.js

```diff
@@ -17,26 +17,61 @@ module.exports = {
         'tutorials/getting-started/initialize-a-data-context',
         'tutorials/getting-started/connect-to-data',
         'tutorials/getting-started/create-your-first-expectations',
-        'tutorials/getting-started/set-up-data-docs',
+        'tutorials/getting-started/check-out-data-docs',
         'tutorials/getting-started/validate-your-data',
         'tutorials/getting-started/customize-your-deployment'
       ]
     },
+    {
+      type: 'category',
+      label: 'Core Skills',
```

Comment:
The way I read the design spec (which may be incorrect!) is that the core skills are a category within each section, not separate from each section. Can we sync on this?